packages/enzyme/src/ReactWrapper.js

```diff
@@ -737,7 +749,7 @@ class ReactWrapper {
    */
   parents(selector) {
     return this.single('parents', (n) => {
-      const allParents = this.wrap(this.single('parents', n => nodeParents(this, n)));
```

Comment:
why is the `single` removed here?

Reply:
This is already being wrapped on the line before

Reply:
`Single` wrapped I mean

Reply:
oh, duh, good catch